nephio-ansible-install/README.md

```diff
@@ -0,0 +1,154 @@
+
+```bash
+mkdir -p inventory
+touch inventory/nephio.yaml
+```
+
+Open an editor of your choice and paste the below in the inventory/nephio.yaml file.
+
+--> You can choose between github repos (remote) or gitea repos (local) by setting the respective vars in the inventory/nephio.yaml file (see below).
+
+```yaml
+all:
+  vars:
+    cloud_user: <username that is used to access the VM>
+    github_username: <github username>
+    github_token: <github personal access token>
+    github_organization: <github organization or username>
+    gitea_username: <gitea username>
+    gitea_password: <gitea password>
+    proxy:
+      http_proxy: 
+      https_proxy:
+      no_proxy:
+    host_os: "linux"  # use "darwin" for MacOS X, "windows" for Windows
+    host_arch: "amd64"  # other possible values: "386","arm64","arm","ppc64le","s390x"
+    tmp_directory: "/tmp"
+    bin_directory: "/usr/local/bin"
+    kubectl_version: "1.25.0"
+    kubectl_checksum_binary: "sha512:fac91d79079672954b9ae9f80b9845fbf373e1c4d3663a84cc1538f89bf70cb85faee1bcd01b6263449f4a2995e7117e1c85ed8e5f137732650e8635b4ecee09"
+    kind_version: "0.17.0"
+    cni_version: "0.8.6"
+    kpt_version: "1.0.0-beta.23"
+    multus_cni_version: "3.9.2"
+    nephio:
+      install_dir: nephio-install
+      packages_url: https://github.com/nephio-project/nephio-packages.git
+    clusters:
+      mgmt: {mgmt_subnet: 172.88.0.0/16, pod_subnet: 10.196.0.0/16, svc_subnet: 10.96.0.0/16}
+      edge1: {mgmt_subnet: 172.89.0.0/16, pod_subnet: 10.197.0.0/16, svc_subnet: 10.97.0.0/16}
+      edge2: {mgmt_subnet: 172.90.0.0/16, pod_subnet: 10.198.0.0/16, svc_subnet: 10.98.0.0/16}
+      region1: {mgmt_subnet: 172.91.0.0/16, pod_subnet: 10.199.0.0/16, svc_subnet: 10.99.0.0/16}
+  children:
+    vm:
+      hosts:
+        <ip address of the VM>:
+```
+
+Some customizations are required to tailor the installation to your environment. Edit the inventory/nephio.yaml file where you update:
+
+- cloud_user: the username that is created to access the VM using SSH
+- github_username: your github user name
+- github_token: github access token to access github [github personal access token](https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+- github_organization: (optional) if you use a github organization for the repo's you should add your github organization here, otherwise it uses the github username
+- gitea_username: your [gitea](https://gitea.io) (local repo) username
+- gitea_password: your [gitea](https://gitea.io) (local repo) password
+
+Note: You can choose between using remote github repos or local gitea repos for your Nephio environment by setting either the github variables or the gitea variables.
+
+To start running ansible playbooks an ansible environment is required. Below is an example how to install ansible using a virtual environment. The repo scripts rely on the ansible galaxy community collection
```

Comment:
Alternatively, ansible playbooks can be mounted inside of an ansible container and executed from there 

e.g; `sudo podman run --name ansible --net=host --cap-add=NET_ADMIN -it -v <path to ansible playbook folder>:/ansible/ -v ~/.kube:/root/.kube -v ~/.ssh:/root/.ssh willhallonline/ansible:2.11.10-centos-7 /bin/sh`

Reply:
yes this is one option we should provide and maybe add even a vagrant/packer file. Some people even suggested docker in docker for this.